packages/docs/src/pages/ui-elements/virtual-scroller.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <docs-content :config="configComputed" />
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
```

Comment:
Let's not use class component (for new stuff at least)

I added and issue to remove it for good - #2345